pylinalg/func/matrix.py

```diff
@@ -134,36 +133,36 @@ def matrix_make_rotation_from_euler_angles(
     -------
     ndarray, [4, 4]
         Rotation matrix.
+
+
+    Notes
+    -----
+    If you are familiar with TreeJS note that this function uses ``order`` to
+    denote both, the order in which rotations are applied *and* the order in
```

Comment:
Nitpick: out-of-place comma
```suggestion
    denote both the order in which rotations are applied *and* the order in
```